episodes/episodes.Rmd

```diff
@@ -598,7 +581,24 @@ Table: Four fruits with color and price in imaginary dollars
 | orange | :orange_square:  | \$3.09   |
 | devil  | :purple_square:  | \$666.00 |
 
-You can see that we now have a caption associated with The table
+### Table alignment best practises
+
+The colons on each side of the `-` in the table dictate how the column is
+aligned. By default, columns are aligned left, but if you add colons on either
+side, that forces the alignment to that side.
+
+In general, most table contents should be left-aligned, with a couple of 
+exceptions:
+
+ - numbers should be right aligned
+ - symbols, emojis, and other equal-width items may be center-aligned
+
+These conventions make it easer for folks to scan a table and understand its
+contents at a glance. 
+
+::::::::::::::::::::::
+
+You can see that we now have a caption associated with the table.
```

Comment:
> You can see that we now have a caption associated with the table.

This sentence should still appear above the callout as a weird standalone sentence/paragraph.